R/mod_marg.R

```diff
@@ -35,9 +36,20 @@
 #' mod <- glm(outcome ~ distance + factor(sex),
 #'            data = margex, family = 'binomial')
 #' mod_marg2(mod, var_interest = 'sex', type = 'levels', at = NULL)
+#'
+#' data(margex)
+#' data(cvcov)
+#' v <- cvcov$ols$clust
+#' d <- cvcov$ols$stata_dof
+#' mod <- glm(outcome ~ treatment + distance,
+#'            data = margex, family = 'binomial')
+#' mod_marg2(mod, var_interest = 'treatment', type = 'levels',
+#'           vcov_mat = v, dof = d)
+
 mod_marg2 <- function(mod, var_interest,
                       type = 'levels',
-                      vcov_mat = vcov(mod),
+                      vcov_mat = NULL,
+                      dof = NULL,
```

Comment:
Made this change (and lines 75-79 below) because otherwise, to test if the user specified a variance-covariance matrix, you have to call vcov(mod) again and check for equality with vcov_mat.